modules/shared/src/main/scala/org/tessellation/ext/codecs.scala

```diff
@@ -41,4 +44,18 @@ object codecs {
         NonEmptySet.fromSetUnsafe(SortedSet.from(nes.toSortedSet))
       }
   }
+
+  object Base64Codec {
+    implicit def decoder[F[_]: Base64, A <: AnyRef: ClassTag: Decoder]: EntityDecoder[F, A] =
+      EntityDecoder.text[F].map { text =>
+        val jsonString = new String(
```

Comment:
why new String is required? It looks like a plain java :D

Reply:
Changed to match PR https://github.com/Constellation-Labs/tessellation/pull/592